pkgs/applications/logging/humioctl/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildGoModule, fetchFromGitHub, installShellFiles, stdenv }:
+
+let
+  version = "0.25.0";
+  sha256 = "1x8354m410nf9g167v0i1c77s5w2by7smdlyjwl89ixgdjw04ay3";
+in buildGoModule {
```

Comment:
Please inline these, there's no point to define these as local variables. You'll have to use `rec` in order to define and use the version in the same derivation FYI.

Reply:
I like having these at the top so that it's immediately clear which variables need to be updated for each update. Furthermore, I've come to see `rec` as an [anti-pattern](https://nix.dev/anti-patterns/language.html#rec-expression). I have, however, made the other suggested changes.